server/readme.md

```diff
@@ -1,33 +1,59 @@
-#### How to set up server-side app
+# Server
+## Pre-Setup -- Local Build
 
-1. Download and install Node which can be done here: https://nodejs.org/en/
+1. Download and install [Node.js and npm](http://nodejs.org/en/) which can be done here: [nodejs.org](http://nodejs.org/)
 2. Setup Node Firebase API with the commands:
 
-   ```
-  $npm install firebase --save
-  $npm install firebase-token-generator --save
-  $npm install firebase-tools
-  ```
-3. Go to your project page in the firebase website. Download firebase service account json object from <your firebase app>/Settings/Permissions/Service Accounts/Create service account' and place it in same directory as server. 
+```bash
+npm install firebase --save
+npm install firebase-token-generator --save
+npm install firebase-tools --save
+```
 
-   *Make sure to select 'Furnish a new private key' and select the key type as JSON, and leave 'Enable Google Apps Domain-wide Delegation' unselected.*
-4. Paste Testing data feature and label files (found in the data folder) into a folder labelled 'DataFiles' and place it in the Server directory.   
-5. Set server and gradient constant values in Constants.js file
-6. Download crowdML-server.js 
-7. Run command
+3. The local environment is now setup proceed to **step 4**
 
-  ```
-  $nodejs crowdML-server.js [Constants file name you wish to use]
-  ```
+## Pre-Setup -- Docker
 
-#### What the server-side app does
+**NOTE**: One may not be able to run both Docker and an Android/iOS emulator. Consider using on device builds with the Docker container. 
```

Comment:
I tried to show that with the headers we can talk in class. See the rendered view https://github.com/3ygun/Crowd-ML/tree/setup-docs/server